sources/web/datalab/noCacheContent.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2015 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
```

Comment:
I think if we put this in a "sources/web/common/" directory, change the import statements to be "import noCacheContent = require('../web/noCacheContent')", and add a "./common/*.ts" argument to the "tsc" invocations, then we will only need one copy of this file. (We could conceivably do that with "common.d.ts" too, but I feel less strongly about that).

Give that a try, and if it doesn't work for some reason let me know.